mypy/newsemanal/semanal_main.py

```diff
@@ -47,9 +47,6 @@
 Patches = List[Tuple[int, Callable[[], None]]]
 
 
-# Perform up to this many semantic analysis iterations until giving up trying to bind all names.
-MAX_ITERATIONS = 10
-
```

Comment:
I think we should keep this, but if we ever hit this, just immediately fail with `AssertionError`, without any additional final pass. Just to avoid infinite loops.